### ISSUE TITLE
[Feature] Ability to set Dark Mode only

### ISSUE THREAD

**Issue by Bortus-AI** [MauriceNino/dashdot#1037]
### Description of the feature

I saw that with the widgets you can specify light or dark theme but did not see that env flag for when launching dashbot via docker or from source.

I would like to be able to have it default to dark or disable light all together

**Comment by MauriceNino** [comment]
Makes sense, I can add a configuration variable, similar to [`DASHDOT_CPU_CORES_TOGGLE_MODE`](https://getdashdot.com/docs/configuration/cpu#dashdot_cpu_cores_toggle_mode).

**Comment by Bortus-AI** [comment]
@MauriceNino That would be awesome. My light sensitivity thanks you haha
